src/dune_rules/coq_rules.ml

```diff
@@ -77,11 +85,17 @@ module Bootstrap = struct
       | false -> Bootstrap lib
       | true -> Bootstrap_prelude)
 
-  let flags =
+  let boot_lib_flags ~coqdoc lib =
+    let open Command in
+    let dir = Coq_lib.src_root lib in
+    if coqdoc then [ Args.A "--coqlib"; Args.Path (Path.build dir) ]
+    else Args.A "-boot" :: Util.theory_coqc_flag lib
+
+  let flags ~coqdoc =
```

Comment:
What about calling this `get_flags`, turning`coqdoc` into `?(coqdoc = false)` and adding a unit argument. That way everybody else should be doing `Bootstrap..get_flags ()`.

Reply:
What's the point of the rename? I don't care. I don't like optional arguments, they are a source of bugs.

Reply:
Actually the `get_foo` style is not used in our source code, so this would look weird.